x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_flights/ml/mean_delay_by_dest_weather.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "groups": [],
```

Comment:
As above, I think there could be some benefit in putting the jobs from the sample modules into a common group `kibana_sample_data`.